core/src/main/java/org/frankframework/util/flow/MermaidFlowGenerator.java

```diff
@@ -93,11 +93,17 @@ public void afterPropertiesSet() throws Exception {
 		transformerPoolConfig = TransformerPool.getInstance(xsltSourceConfig, 2);
 	}
 
-	private String compileFrankElementList() throws SAXException, ClassNotFoundException {
+	private String compileFrankElementList() throws SAXException {
 		try (SaxDocumentBuilder builder = new SaxDocumentBuilder("root")) {
 			List<String> classNames = findAllFrankElements();
 			for(String className : classNames) {
-				Class<?> clazz = Class.forName(className);
+				Class<?> clazz;
+				try {
+					clazz = Class.forName(className);
+				} catch (ExceptionInInitializerError | NoClassDefFoundError | Exception e) {
+					log.info("Found class [{}] on classpath which cannot be loaded due to exception", className, e);
```

Comment:
This is not a serious message that should warn an user / integration specialist?

Reply:
I don't think so. I think that if you know you do not have SAP-CD libraries on your classpath but you have that module loaded because it's just part of the bundle, you do not need to see that as a warning every time you start up the system.

Reply:
Ok thanks. I just didn't know how this is used in practice.

Reply:
This is just for the flow visualization, in fact, only for the colors or Frank!Elements 😄